repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/ServiceClient.java

```diff
@@ -80,7 +79,7 @@ public ServiceClient(String targetHostUri, String username, String password, Str
     }
 
     private HttpClient getClientWithBasicAuth() throws ServiceClientException {
```

Comment:
Is throwing this exception still necessary?  If not, we may not need the exception class anymore.